website/content/commands/docs.mdx

```diff
@@ -17,6 +17,11 @@ Show documentation for components
 
 Usage: `waypoint docs [options]`
 
+Output documentation about the plugins. By default, it lists the documentation
+for the plugins configured by this project.
+
+    The flags can change which plugins are listed and in which format.
+
```

Comment:
This indented line shows up in a code block:
<img width="1065" alt="Screen Shot 2021-08-10 at 9 47 29 AM" src="https://user-images.githubusercontent.com/8404559/128878442-e11cc6d4-1920-4007-b8c8-4b43f698db4f.png">